composer.json

```diff
@@ -1,9 +1,9 @@
 {
-    "name": "islandora/claw-jsonld",
+    "name": "islandora/jsonld",
```

Comment:
if name changes then @dannylamb needs to update https://github.com/Islandora-CLAW/islandora/pull/53/files/64e4386bfa1c5e98b9b63cf37534a5bf600c4843..9686c9054d3fe8c3fce54b4414e794b37035f7f5#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R21. Also other modules (not sure which ones) need their composer updated if they depend on this package